test.config

```diff
@@ -0,0 +1,27 @@
+params {
+  outdir = "result_test"
+  // input for hashing-based deconvolution
+  hto_matrix_raw = "$projectDir/test_data/hto"
```

Comment:
Ideally all of these should eventually be fetched directly from the cloud but let's not bother with that for now. Downloading them with the script is OK.